packages/fuel-indexer-lib/src/manifest.rs

```diff
@@ -46,6 +47,31 @@ impl ToString for Module {
     }
 }
 
+/// in a `Manifest` struct. It can either be a single `Option<String>`
+/// or a `Vec<Option<String>>`.
+///
+/// - `Single`: Represents an optional single contract ID as an `Option<String>`.
+/// - `Multiple`: Represents a vector of optional contract IDs as a `Vec<Option<String>>`.
```

Comment:
Can you change this so that its first line is consistent with the other doc comments?

Reply:
Yep!